webpack.config.js

```diff
@@ -36,12 +35,12 @@ const config = {
       },
       {
         test: /\.(js|jsx)$/i,
-        loader: "babel-loader",
+        exclude: /(node_modules)/,
```

Comment:
Nitpick: brackets are not needed

Reply:
adjusted in 5aae9baf51ec19db59531588614bd3f8c7aee519